airflow/requirements.txt

```diff
@@ -6,7 +6,7 @@ gusty==0.6.0
 pandas-gbq==0.14.1
 pybigquery==0.7.0
 google-cloud-bigquery==2.16.1
-calitp==0.0.8
+calitp==0.0.12
```

Comment:
I believe a number of airflow items rely on the calitp library things and some dependencies referenced by calitp-py. Can we make sure that the airflow DAGs run ok with this bump?

Reply:
Cauley reverted; we don't need this change in Airflow